src/server/scripts/Northrend/Ulduar/Ulduar/ulduar.h

```diff
@@ -368,7 +399,7 @@ enum UlduarData
     DATA_HODIR_YS,
     DATA_THORIM_YS,
     DATA_MIMIRON_YS,
-    DATA_ILLUSION,
+    DATA_ILLUSION, // = 46 used by conditions
```

Comment:
it could break something if anyone add an enum before this one (multiple enums with same value)
what do you think about `static_assert(DATA_ILLUSION == 46)` in instancescript?